js/bitz.js

```diff
@@ -40,9 +40,9 @@ module.exports = class bitz extends Exchange {
             'urls': {
                 'logo': 'https://user-images.githubusercontent.com/1294454/35862606-4f554f14-0b5d-11e8-957d-35058c504b6f.jpg',
                 'api': {
-                    'market': 'https://apiv2.bitz.com',
-                    'trade': 'https://apiv2.bitz.com',
-                    'assets': 'https://apiv2.bitz.com',
+                    'market': 'https://apiv2.bitz.pro',
```

Comment:
Hey) Thx! One question through, shouldn't it be `https://apiv2.bit-z.pro`, no? `bit-z` vs `bitz` )

Reply:
oh my fault.